lib/fhir/root.js

```diff
@@ -271,8 +272,9 @@ module.exports = (mongo, modules, callback) => {
     searchAll: (ctx) => {
       throw new Error('Not yet supported')
     },
-    conformance: (ctx) => {
-      throw new Error('Not yet supported')
+    conformance: (ctx, callback) => {
+      logger.info(`Fetching conformance statement`)
```

Comment:
I would think that logging this at info level is a bit too noisy. Can this be reduced to debug instead?